DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/BasicConnectorBinding.cs

```diff
@@ -88,35 +88,63 @@ private async void HighlightObjectsOnView(List<string> objectIds)
     await QueuedTask
       .Run(() =>
       {
-        List<MapMember> mapMembers = GetMapMembers(objectIds, mapView);
+        Dictionary<string, (MapMember member, List<int>? rows)> mapMembersRows = GetMapMembers(objectIds, mapView);
         ClearSelectionInTOC();
         ClearSelection();
-        SelectMapMembersInTOC(mapMembers);
-        SelectMapMembers(mapMembers);
+        SelectMapMembersInTOC(mapMembersRows.Values.ToList().Select(x => x.member).ToList());
+        SelectMapMembers(mapMembersRows);
         mapView.ZoomToSelected();
       })
       .ConfigureAwait(false);
   }
 
-  private List<MapMember> GetMapMembers(List<string> objectIds, MapView mapView)
+  private Dictionary<string, (MapMember member, List<int>? rows)> GetMapMembers(List<string> objectIds, MapView mapView)
   {
-    List<MapMember> mapMembers = new();
+    Dictionary<string, (MapMember member, List<int>? rows)> mapMembersRows = new();
 
     foreach (string objectId in objectIds)
     {
-      MapMember mapMember = mapView.Map.FindLayer(objectId);
+      List<string> uriRows = objectId.Split("_speckleRowIndex_").ToList();
+      string mapURI = uriRows[0];
```

Comment:
Once we split `objectId` with magical keyword, is the first one mapURI? Is this mean we combined them before with `_speckleRowIndex_`? We should make a class to handle this if we do some magic here.